src/pages/SNSPage.jsx

```diff
@@ -0,0 +1,53 @@
+import { css } from '@emotion/react';
+import { useEffect, useState } from 'react';
+
+import Header from '../components/header/Header.jsx';
+import Content from '../components/content/Content.jsx';
+import Footer from '../components/footer/Footer.jsx';
+import mockData from '../mock/mockData.json';
+
+const pageWrapper = css`
+  position: relative;
+  margin: 0 auto;
+  /* overflow: hidden; */
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  max-width: 430px;
+  border: 1px solid var(--border-color);
+`;
+
+const otherNickName = '손오공';
+const otherName = 'svt';
+const currentName = 'carat';
+
+export default function SNSPage() {
+  const [messages, setMessages] = useState(() => {
+    const mockDataList = mockData;
+    return mockDataList? mockDataList : [];
+  });
```

Comment:
`json`에 저장된 내용을 불러오고 싶을 때에는 아래처럼 간단하게 작성할 수 있습니다!
```suggestion
  const [messages, setMessages] = useState(mockData);
```